src/main/java/org/helpberkeley/memberdata/WorkflowBean.java

```diff
@@ -47,6 +47,7 @@ public interface WorkflowBean {
     String getControlBlockKey();
     String getControlBlockValue();
     String getGMapURL();
+    String[] getMemberData();
```

Comment:
I get why it's too opaque and since I'm only using it once it makes sense to call the accessors directly in the calling code, but I don't get why a list would be better than an array here. Can you explain why a List is preferred in this case?

Reply:
A list here is only marginally better, in that it is not an array.  Interfaces/Classes in the Collections framework offer a variety of methods for management and manipulation that arrays don't.  Easily adding/removing elements.  Various types of iteration and search. Generics support.  Lambda support.

Reply:
Another take on this would be to return a Map, keyed by the column name.  The Map interface would give you the ability to iterate by the keys and the values.  But overall, using the original accessors directly is a  better choice.